packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -16,6 +16,7 @@ public struct MediaInfo {
 public enum Capabilities: String {
     case mentions
     case unsupportedBlockEditor
+    case unsupportedBlockEditorSwitch
```

Comment:
From what I see [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/14880/files#diff-ae5d7f8fb9cae1bc2a433c8133c06674R943), `unsupportedBlockEditorSwitch` is true if Jetpack is connected with the site, false otherwise. 

Could we change the name of this to be more descriptive?

Reply:
I understand the need to avoid adding Jetpack-specific terms here, but at the moment it feels ambiguous. Maybe `unsupportedBlockEditorCustomAction` or something like that.